Source/ARTCrypto.h

```diff
@@ -44,11 +51,30 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+/**
+ The ARTCrypto object ensures that message payloads are encrypted, can never be decrypted by Ably, and can only be decrypted by other clients that share the same secret symmetric key.
+ */
 @interface ARTCrypto : NSObject
 
-+ (ARTCipherParams *)getDefaultParams:(NSDictionary *)values;
+/**
+ Retrieves, or optionally sets, the `ARTCipherParams` for the channel.
+ @param params Overrides the default parameters. A suitable key must be provided as a minimum.
+ @return `ARTCipherParams` A ARTCipherParams object, using the default values for any field not supplied.
```

Comment:
I think it should be "**An** ARTCipherParams object", not "A".

Reply:
https://github.com/ably/ably-cocoa/pull/1439/commits/3e0875b98ece697f3b01cc346f3b803c886fa423